### ISSUE TITLE
grid_search_forecaster_multiseries - error 

### ISSUE THREAD

**Issue by AVPokrovsky** [JoaquinAmatRodrigo/skforecast#714]
Hi, I upgraded to the last version and got this error from grid_search_forecaster_multiseries. It worked with previous version with the same data (I don't remember which version was used before)

210 models compared for 1 level(s). Number of iterations: 210.
lags grid:   0%|          | 0/7 [00:00<?, ?it/s]
params grid:   0%|          | 0/30 [00:00<?, ?it/s]
lags grid:   0%|          | 0/7 [00:00<?, ?it/s]TypeError: DataFrame.reset_index() got an unexpected keyword argument 'names'

**Comment by AVPokrovsky** [comment]
It seems this is not a reason. I already have 1.5.1

Name: pandas
Version: 1.5.1
Summary: Powerful data structures for data analysis, time series, and statistics
Home-page: https://pandas.pydata.org
Author: The Pandas Development Team
Author-email: pandas-dev@python.org
License: BSD-3-Clause
Location: c:\anaconda\envs\test7\lib\site-packages
Requires: python-dateutil, numpy, pytz
Required-by: yfinance, yahoo-fin, trendln, streamlit, statsmodels, spacy-streamlit, skforecast, seaborn, Quandl, pytrends, prophet, pmdarima, pandas-ta, pandas-datareader, cmdstanpy, catboost, altair

**Comment by JavierEscobarOrtiz** [comment]
This is strange, looking at the pandas documentation, `names` was introduced in this version:

https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.reset_index.html

![imagen](https://github.com/JoaquinAmatRodrigo/skforecast/assets/95030312/6dc362fc-c9c3-4b71-8e71-b78a37092527)

Could you please try with pandas 2.0?

**Comment by JavierEscobarOrtiz** [comment]
I open a new issue regarding the RNN regressor @FernandoCarazoMelo

**Comment by bassamabdelghani** [comment]
> Finally I resolved this issue changing pandas version. I got another one with RNN regressor (below). It also appears with you code examples on RNN. It was resolved when I downgraded Tensorflow from 2.16.1 to 2.15.1
> 
> Input shape of the regressor should be Input(shape=(lags, n_series))

how you managed to install the older version 2.15.1? 
when i tried to do that i gett the below error :

ERROR: Could not find a version that satisfies the requirement tensorflow==2.15.1 (from versions: 2.16.0rc0, 2.16.1, 2.16.2, 2.17.0rc0, 2.17.0rc1, 2.17.0)
ERROR: No matching distribution found for tensorflow==2.15.1

**Comment by FernandoCarazoMelo** [comment]
Hi, @bassamabdelghani , 

Which Python version are you using?

**Comment by bassamabdelghani** [comment]
> Hi, @bassamabdelghani ,
> 
> Which Python version are you using?

I'm using version 3.12.4
